src_c/pgcompat.c

```diff
@@ -0,0 +1,43 @@
+#include <SDL.h>
+
+#if defined(SDL_VERSION_ATLEAST)
+#if !(SDL_VERSION_ATLEAST(2, 0, 5))
+/* this function is only defined in 2.0.5.
+  https://wiki.libsdl.org/SDL_SetWindowResizable
+*/
+void SDL_SetWindowResizable(SDL_Window* window,
+                            SDL_bool    resizable) {
+
+}
+int SDL_GetWindowOpacity(SDL_Window* window,
+                         float*      opacity) {
+    SDL_SetError("SDL 2.0.5 required!");
```

Comment:
Changing the error text (and all the other similar ones) from `"SDL 2.0.5 required!"` to something like `"SDL 2.0.5 or greater required"` might be more accurate.